clojure-mode.el

```diff
@@ -2142,23 +2142,30 @@ DIRECTION is `forward' or `backward'."
               (setq candidate (string-remove-prefix "'" (thing-at-point 'symbol))))))))
     candidate))
 
-(defun clojure-find-ns ()
-  "Return the namespace of the current Clojure buffer.
+(defun clojure-find-ns (&optional supress-errors)
```

Comment:
It's actually "suppress" (double p). Btw, on second thought "ignore-errors" might be an even better name, but I'm fine with both options.

Reply:
Arrgh :)

Amended. I left it at suppress because there's already a construct named `ignore-errors` (probably Elisp would have been fine with it. Humans not so much?)